### PR TITLE
Draw chessboard

### PR DESCRIPTION
**Pull request by mbrown78**
Pieces appear on board once it is initialized. Pieces images were created with html unicode. Tests still need to be completed.

### CODE REVIEW

**Comment by saakuffo** [comment]
Hey Melissa, like we discussed, this branch would have to be rebuilt after master committed the changes. Almost all 'files changed' are now redundant. It would probably be cleaner to just pull current master and put in your current actual changes.

**Comment by lasley** [comment]
The way to do this is to rebase:

```
git checkout master
git pull
git checkout Draw_Chessboard
git rebase master
```

**Comment by mbrown78** [comment]
Test added, new master has been pulled in.

**Comment by lasley** [comment]
I forgot to give the push command. You should not pull the branch after rebasing

```
git checkout master
git pull
git checkout Draw_Chessboard
git rebase master
git push -f
```

**Comment by lasley** [comment]
Congrats on your first successful rebase @mbrown78 

Will review this sometime today

File: app/models/game.rb [hunk 1]
```diff
@@ -1,54 +1,68 @@
...
         y_position: 1,
-        color: 'white'
+        color: 'white',
+        image: '&#9817;',
       )
     end
 
-    Rook.create(game_id: id, x_position: 0, y_position: 0, color: 'white')
-    Rook.create(game_id: id, x_position: 7, y_position: 0, color: 'white')
+    Rook.create(game_id: id, x_position: 0, y_position: 0, color: 'white', image: '	&#9814;')
```
Comment (lasley) [inline app/models/game.rb]: The image should be defined in the appropriate piece models as a method returning either the black or white image for the piece

File: config/application.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Add configuration values here, as shown below.
```
Comment (lasley) [inline config/application.yml]: This file is not related

**Comment by mbrown78** [comment]
all working now

File: config/application.yml (hunk 2 above)
Comment (lasley) [inline config/application.yml]: This has not been attended to. You should also regenerate the Facebook secret in your Facebook account - there are numerous bots that have a specific purpose to monitor public repos scraping keys for spam use.

File: app/views/games/show.html.erb [hunk 3]
```diff
@@ -1 +1,13 @@
 <h1><%= @game.game_title %></h1>
+
+<div class="chessboard">
+  <table id="chessboard_table" >
+      <% 7.downto(0) do |row| %>
+        <tr class= "chessboard-col">
+          <% 7.downto(0) do |col| %>
+              <td id="chess_piece"><%=@game.return_piece(col,row)%></td>
```
Comment (lasley) [inline app/views/games/show.html.erb]: You can't have multiple elements with the same `id` attribute. I think maybe you were going for the `name` attribute here instead? 

This would be useful if you ever want to use jQuery to quickly select all piece tds - `$('td[name="chess_piece"]')` but IMO is ambiguous because it's really a chess square. It's also best practice to use the class attribute for that.

Additionally, it well help with the AJAX side of things if you do use the `id` attribute, but instead put the piece id in it. This way you can `$('#piece_id')` to get whatever piece you need.

And final additionally - front end (HTML, Javascript, CSS) styling convention is to use `lowerCamelCase` in places where we would normally use `lower_snake_case` in Rails/Ruby. Eg this would be `chessPiece`

**Comment by mbrown78** [comment]
application.yml file removed. The CSS, HTML and JS naming conventions are changed in the future branch move_pieces_js that will be pushed. thanks for the ajax tip

**Comment by lasley** [comment]
@mbrown78 - Alright so to confirm, the remaining of my comments are to be pushed back into the AJAX moving PR?

**Comment by mbrown78** [comment]
Yes

**Comment by saakuffo** [comment]
Conflicts need to be resolved.
This is failing tests.

**Comment by mbrown78** [comment]
these issues have been resolved in move_pieces_js

**Comment by lasley** [comment]
@mbrown78 - I dislike what just happened here. I have reviewed >400 lines of codes spanning 18 files. Moving this into another PR makes me re-review all of that, which is horribly bad practice. Why are we not continuing the changes here?
